ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/baselinecaps/vcurrent/provision/cucushift-installer-rehearse-gcp-upi-baselinecaps-vcurrent-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-baselinecaps-vcurrent-provision-chain
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
+  - ref: upi-install-gcp
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource
```

Comment:
If it is vcurrent baselinecaps, will olm operators be enabled or not? if not, I guess we do not need enable-qe-catalogsource